### PR TITLE
Added Functionality to ImageWidget

### PR DESCRIPTION
**Pull request by spoorthyv**
The image widget now starts of fit to width instead of stretching to fill the whole screen (I couldn't get it to fit to both width and height). We also added zoom in and out buttons in the command palette that apply to all open images.

We are still trying to figure out how to only apply zooming on the active widget.

@charnpreetsingh185

### CODE REVIEW

File: src/imagewidget/plugin.ts [hunk 1]
```diff
@@ -28,15 +34,95 @@ export
...
+      },
+    ]);
+
+    function zoomIn(): void {
+        if (scale > 1) {
+            scale += .5;
+        } else {
+            scale *= 2;
+        }
+        let widgets = tracker.widgets;
```
Comment (blink1073) [inline src/imagewidget/plugin.ts]: You can use tracker.activeWidget to affect only the active widget

**Comment by spoorthyv** [comment]
That almost works for me. Small bug though. 
When I open an image, it isnt considered active until I click on the window.
ezgif-1740586634

**Comment by blink1073** [comment]
You can manually focus the node after adding it to the tracker for now.

**Comment by spoorthyv** [comment]
I think the image has focus right after adding it. It just loses focus as soon as you click the command palette

**Comment by spoorthyv** [comment]
Looks like it passed now

**Comment by blink1073** [comment]
Works well, thank you!

**Pull request merged**